src/OrbitBase/NotFoundOrTest.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest-death-test.h>
```

Comment:
This is not a public header. I think you can just remove the include and then copybara shouldn't fail anymore.